include/dlr_tvm.h

```diff
@@ -13,7 +19,8 @@ ModelPath GetTvmPaths(std::vector<std::string> tar_path);
 
 /*! \brief class TVMModel
  */
-class TVMModel : public DLRModel {
+
+class  DLR_DLL TVMModel : public DLRModel {
```

Comment:
Fix double space and extra newline

Reply:
Perhaps run `clang-format -style="google"`

Reply:
Good catch, fixed.